src/electionguard_cli/cli_models/e2e/e2e_inputs.py

```diff
@@ -2,10 +2,14 @@
 from electionguard.ballot import PlaintextBallot
 from electionguard.guardian import Guardian
 from electionguard.manifest import Manifest
+from electionguard_cli.cli_models.shared.cli_election_inputs_base import (
+    CliElectionInputsBase,
+)
 
 
-class E2eInputs:
-    """Responsible for holding the inputs to an end-to-end election."""
+# pylint: disable=too-many-instance-attributes
```

Comment:
This seems to be a very common lint rule to disable, maybe we should do it globally?

Reply:
We shouldn't. This is really strong code smell. We should be making adjustments to avoid it. For example, this should be a `dataclass`.

Reply:
We are currently disabling too-many-instance-attributes it in 12 other places in the code base.  I actually tried using the dataclass, but it wasn't smart enough to handle inheritance, I kept getting `TypeError: __init__() takes 5 positional arguments but 9 were given`.  I did a bit of research and it looks like Python 3.10 solves this with the `kw_only=true` dataclass attribute parameter: https://medium.com/@aniscampos/python-dataclass-inheritance-finally-686eaf60fbb5.  Any other thoughts on what I should be doing here?